jest.setup.js

```diff
@@ -1,10 +1,34 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
 // TODO [#339]: replace third-party libraries with NPM modules
 import jQuery from './_dprhtml/js/external/jquery-3.4.1.min';
 import knockout from './_dprhtml/js/external/ajax/libs/knockout/3.5.0/knockout-min';
 
+window.$ = jQuery;
 window.jQuery = jQuery;
 window.ko = knockout;
 
 window.DPR_G = require('./_dprhtml/js/globalObject');
 window.DPR_PAL = require('./_dprhtml/js/dpr_pal');
+window.DPR_translitCore_mod = require('./_dprhtml/js/translitCore');
+window.DPR_translit_mod = require('./_dprhtml/js/translit');
+window.DPR1_chrome_mod = require('./_dprhtml/js/chrome').DPR1_chrome_mod;
```

Comment:
Some of these files have multiple global "exports". For example, `./_dprhtml/js/chrome.js` defines `DPR1_chrome_mod` and `DPR_Chrome`.